docs/devel/translations.rst

```diff
@@ -17,10 +17,42 @@ This can be done using :guilabel:`Start new translation` in :ref:`component`.
 
 .. note::
 
-    Project admins can always start translation within Weblate directly.
+    If you add a language file in connected remote repository, respective
+    translation will be added to the component when Weblate updates local repository.
+     
+    More info on the repository update settings can be found on the :ref:`update-vcs`.
+
+.. _removing-translation:
+
+Removing existing translations
+------------------------------
+
+Languages, components, or the projects they are in, can be removed (deleted from Weblate
+and remote repository if used) from the menu :guilabel:`Manage` ↓ :guilabel:`Removal`
```

Comment:
is arrow pointing down intentional here?

Reply:
Yes, it is a dropdown.